app/models/nearby_facility.rb

```diff
@@ -7,13 +7,44 @@ class << self
     attr_writer :validate_on_load
 
     def query(params)
-      json_from_file = File.read(Rails.root.join('modules', 'va_facilities', 'nearby.json'))
-      mocked_response = JSON.parse(json_from_file)
-      return mocked_response if params[:street_address] && params[:city] && params[:state] && params[:zip]
+      if params[:street_address] && params[:city] && params[:state] && params[:zip]
+        isochrone_response = request_isochrone(params)
+        return get_facilities_in_isochrone(params, isochrone_response)
+      end
 
       NearbyFacility.none
     end
 
+    def request_isochrone(params)
+      params[:drive_time] = '30' unless params[:drive_time]
+      address = "#{params[:street_address]} #{params[:city]} #{params[:state]} #{params[:zip]}"
+      query = {
+        waypoint: address,
+        maxtime: params[:drive_time],
+        timeUnit: 'minute',
+        dateTime: '07:30:00',
```

Comment:
Is this a hard coded date? What is this? Maybe write some comments about the params that the bing api requires, or maybe just comment with a link to the api docs or something?

Reply:
Good point. I will add a comment to link to the documentation and give a bit of background on how we are handling traffic modelling.

`dateTime: '07:30:00'` just means that when Bing is modelling traffic it is looking at today at 7:30 AM (local time for the waypoint). This is just our initial attempt at taking traffic into account.